tests/func/test_add.py

```diff
@@ -475,6 +475,7 @@ def test(self):
         self.assertEqual(0, ret)
 
         foo_stage_file = self.FOO + Stage.STAGE_FILE_SUFFIX
+
```

Comment:
why?

Reply:
I had changed the comment bellow from "stage file" from "DVC-file" but Ivan said to leave comments alone so I reverted it. I also added that new line to match the other similar code block in `class TestShouldThrowProperExceptionOnCorruptedStageFile`.

Reply:
I was refering to the newline itself. It is polluting the git blame :slightly_smiling_face: Let's leave it this time, but please don't leave such things in the future.